modules/accessing-the-pipeline-editor.adoc

```diff
@@ -16,9 +16,13 @@ ifdef::upstream[]
 endif::[]
 
 * You have created a data science project.
-* You have created a workbench with the *Standard Data Science* notebook image.
 * You have created and configured a pipeline server within the data science project that contains your workbench.
-* You have created and launched a Jupyter server from a notebook image that contains the Elyra extension (Standard data science, TensorFlow, TrustyAI, or PyTorch).
++
+[IMPORTANT]
+====
+To ensure the runtime configuration is created automatically, you must have created the pipeline server before you created the workbench.
+====
+* You have created and launched a workbench using a notebook image that contains the Elyra extension (Standard data science, TensorFlow, TrustyAI, or PyTorch).
```

Comment:
```suggestion
* You have created a workbench with a workbench image that contains the Elyra extension (Standard Data Science, TensorFlow, TrustyAI, or PyTorch), as described in https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.16/html/working_on_data_science_projects/using-project-workbenches_projects#creating-a-workbench-select-ide_projects.
* You have started the workbench and opened the JupyterLab interface, as described in https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.16/html/working_in_your_data_science_ide/accessing-your-workbench-ide_ide.
```

We should think about standardising on adding links to all prereqs in all modules, but I've added these two for now because of the confusion reported by a reader.

You'll obviously need to format them correctly using attributes and link text -- and conditionalise for upstream, self-managed, and cloud -- I just included these quick links to let you know the target locations.